cluster/get-kube-binaries.sh

```diff
@@ -119,13 +119,24 @@ function sha1sum_file() {
   fi
 }
 
+# Get default service account credentials of the VM.
+function get-credentials {
+  curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/token" -H "Metadata-Flavor: Google" -s | python -c \
```

Comment:
What OS environment does prow run inside of? Is gcloud installed?

Reply:
prow runs a pod with whatever image user defines, we will ensure to use an image with gcloud for our use case here - also I think optionally use gsutil might be a better idea

Reply:
we don't really need gcloud, it is just a local endpoint in the VM